clients/admin-ui/src/features/system/ManualSystemFlow.tsx

```diff
@@ -45,56 +47,141 @@ const ConfigureSteps = ({
 const ManualSystemFlow = () => {
   const router = useRouter();
   const dispatch = useAppDispatch();
-
   const [currentStepIndex, setCurrentStepIndex] = useState(0);
   const activeSystem = useAppSelector(selectActiveSystem);
+  const { navV2 } = useFeatures();
 
-  const goBack = () => {
+  const goBack = useCallback(() => {
     router.back();
     dispatch(setActiveSystem(undefined));
-  };
+  }, [dispatch, router]);
+
+  const handleSuccess = useCallback(
+    (system: System) => {
+      setCurrentStepIndex(currentStepIndex + 1);
```

Comment:
Something seems to be going wrong with how this value is picked up by the tabs. "Confirm and Continue" submits the form, but it doesn't continue to the next tab: https://www.loom.com/share/84e49d8fcfde437db70ab2394c013f5b

^ I also put in a console log to confirm handleSuccess is called.

Reply:
Issue has been resolved.